packages/kbn-ecs/BUILD.bazel

```diff
@@ -0,0 +1,107 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler", "pkg_npm", "pkg_npm_types", "ts_project")
+
+PKG_DIRNAME = "kbn-ecs"
+PKG_REQUIRE_NAME = "@kbn/ecs"
+
+SOURCE_FILES = glob(
+  [
+    "**/*.ts",
+  ],
+  exclude = [
```

Comment:
It doesnt affect the build but can take out a bunch of these exceptions since this is just type definitions  we would have a `/storybook`, or /snapshots` etc